backend/app/api/auth.py

```
@@ -0,0 +1,152 @@
+"""
+Authentication API endpoints
+"""
+
+from fastapi import APIRouter, HTTPException, Depends
+from fastapi.security import HTTPAuthorizationCredentials
+from datetime import datetime, timedelta
+from typing import Optional
+import uuid
+from app.models.auth import *
```

Ruff (0.13.1)

`from app.models.auth import *` used; unable to detect undefined names

(F403)